data/common/import_.py

```diff
@@ -13,7 +14,6 @@
 
 from common.export import create_activity, delete_exchange, new_exchange, search
 
-BIOSPHERE = "biosphere3"
```

Comment:
because I used to gather the remaining substances of AGB into a specific biosphere but it is of no use. Si I keep the default.  Same for the BW project which is again `default` instead of `food` or `textile` (maybe in another PR)